metaci/testresults/views.py

```diff
@@ -159,9 +161,26 @@ def resolve_asset_url(m):
         asset_id = int(m.group(2))
         try:
             asset = queryset.get(id=asset_id)
-            url = asset.asset.url
         except ObjectDoesNotExist:
```

Comment:
Thanks for sending me down the `foo = foo.filter(id=x).first() --> if foo:...` vs. `try...except` rabbit hole.